client/asset/dcr/dcr.go

```diff
@@ -613,6 +614,11 @@ func (dcr *ExchangeWallet) Balance() (*asset.Balance, error) {
 	}, nil
 }
 
+// FeeRate satisfies asset.FeeRater.
+func (dcr *ExchangeWallet) FeeRate() (uint64, error) {
+	return dcr.feeRate(2)
```

Comment:
Let's make a comment that feeRate(1) produces strangely high results when not necessary